### PR TITLE
Minor errors to get notebook running

### PR DESCRIPTION
**Pull request by al-niessner**

### CODE REVIEW

**Comment by al-niessner** [comment]
@tloubrieu-jpl 

Please review and merge. Seems to be working for me now.

**Comment by jordanpadams** [comment]
@tloubrieu-jpl is this still a WIP? or are we good to merge?

**Comment by tloubrieu-jpl** [comment]
@al-niessner can you look at my comments on the pull request. No a big deal but that will allow to merge the pull request. Thanks.

**Review by tloubrieu-jpl — CHANGES_REQUESTED** [review]

**Comment by tloubrieu-jpl** [comment]
@jordanpadams ok, I missed to complete the review. That is done now.

File: requirements.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ astropy==4.2
 matplotlib==3.3.4
 jupyterlab==3.0.9
 tqdm==4.59.0
-
+ipywidgets
```
[the remark this answers is not in the document]
Comment (al-niessner) [reply requirements.txt]: done

**Review by nutjob4life — APPROVED** [review]
Fine with me; 🚢 it.

**Review by tloubrieu-jpl — APPROVED** [review]

**Pull request merged**